fastlane/Fastfile

```diff
@@ -0,0 +1,179 @@
+
+before_all do
+  setup_circle_ci
+end
+
+desc "Increment build numbers"
+lane :bump_version do |options|
+  if options[:dry_run]
+    dry_run = true
+  end
+  if options[:version]
+    new_version_number = options[:version]
+  else
+    UI.user_error!("Missing `version` argument")
+  end
+
+  # If we chain lanes together, we'll need to make sure we don't chdir out of the correct directory
+  if !File.file?("package.json")
+    Dir.chdir("..")
+  end  
+  previous_version_number = (sh "awk -F\\\" \'/\"version\":\/ {print $4}\' package.json").strip
+  UI.message("ℹ️  Current version: #{previous_version_number}")
+  UI.message("ℹ️  Setting version: #{new_version_number}")
+  files_to_update = [
+    'package.json',
+    'ios/RNPurchases.m',
+    'android/build.gradle',
+  ]
+
+  if dry_run
+    UI.message("ℹ️  Nothing more to do, dry_run: true")
+  else
+    for file_to_update in files_to_update
+      increment_build_number(previous_version_number, new_version_number, file_to_update)
+    end 
+  end
+end
+
+desc "Build example"
+lane :build_example do |options|
+  # If we chain lanes together, we'll need to make sure we don't chdir out of the correct directory
+  if !File.file?("package.json")
+    Dir.chdir("..")
+  end  
+
+  sh("yarn example")
+  Dir.chdir("examples/purchaseTester")
+  sh("npx pod-install")
+end
+
+desc "Prepare for release by bumping verion and building the example."
+lane :prepare_for_release do |options|
+  bump_version(options)
+  build_example(options)
+end
+
+desc "Update hybrid common pod and gradle"
+lane :update_hybrid_common do |options|
+  if options[:dry_run]
+    dry_run = true
+  end
+  if options[:version]
+    new_version_number = options[:version]
+  else
+    UI.user_error!("Missing `version` argument")
+  end
+
+  # If we chain lanes together, we'll need to make sure we don't chdir out of the correct directory
+  if !File.file?("RNPurchases.podspec")
+    Dir.chdir("..")
+  end  
+  previous_version_number=parse_previous_hybrid_version
+  # Add to options so if we're chaining lanes together, the other lanes get it
+  options[:previous_hybrid_version] = previous_version_number
+
+  UI.message("ℹ️  Current Purchases version: #{previous_version_number}")
+  UI.message("ℹ️  Setting Purchases version: #{new_version_number}")
+  files_to_update = [
+    'RNPurchases.podspec',
```

Comment:
updating the podspec can be done with https://docs.fastlane.tools/actions/version_bump_podspec/
See example usage [in our flutter repo](https://github.com/RevenueCat/purchases-flutter/blob/develop/fastlane/Fastfile#L35)

Reply:
This is actually to increase the version of PurchasesHybridCommon, not the version of RNPurchases, which is got from the package.json